src/object.d

```diff
@@ -2678,14 +2683,14 @@ if (is(typeof(create()) : V) && is(typeof(update(aa[K.init])) : V))
     aa.update("k1", {
         return -1; // create (won't be executed
```

Comment:
Missing closing paren in comment.

Reply:
Thanks. https://github.com/dlang/druntime/pull/3014